include/CUFLU.h

```diff
@@ -413,7 +421,11 @@
 
 // 3. dt solver for fluid
 //=========================================================================================
+#  if ( MODEL == HYDRO  &&  EOS == EOS_NUCLEAR )
```

Comment:
Is `If (EOS == EOS_NUCLEAR)` enough?

Reply:
Good suggestion. I'll remove the redundant check.